snap/config/security-gateway-api/res/configuration.toml

```diff
@@ -0,0 +1,93 @@
+#################################################################################
+# Copyright 2018 Dell Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+# @author: Tingyu Zeng, Dell
+# @version: 0.1.0
+#################################################################################
+
+# This is a TOML config file for edgexsecurity service.
+
+title = "EdgeX security service config file"
+
+[kongurl]
+server = "localhost"
+adminport = "8001"
+applicationport = "8000"
+
+[kongadmin]
+username = "administrator"
+password = "changeme"
+
+[secretservice]
+server = "localhost"
+port = "8200"
+healthcheckpath = "v1/sys/health"
+certpath = "v1/secret/edgex/pki/tls/edgex-kong"
+tokenpath = "res/resp-init.json"
```

Comment:
As the only thing that's changed in this file is "tokenpath", wouldn't a better approach be to use sed to modify the source file before copying it into **$SNAP/config/$service/res**?  That way if any changes are made to the original, we'll pick them up in the snap.

Reply:
Sure I can do this as well

Reply:
I have done this even though it is terrifying and makes me feel dirty...
```
sed -i "s@tokenpath = \"res\\\\\\\\resp-init.json\"@tokenpath = \"res/resp-init.json\"@" $SNAPCRAFT_PART_INSTALL/config/security-api-gateway/config/res/configuration.toml
```
8(!) `\` ...